app/assets/stylesheets/partials/_mixins.scss

```diff
@@ -0,0 +1,9 @@
+// Control the reading expreience
+@mixin well-read {
```

Comment:
I like it! Creating reusable mixins will definitely improve our CSS! Thanks for making this partial.